src/applications/yellow-ribbon/containers/YellowRibbonApp/index.jsx

```diff
@@ -71,29 +58,37 @@ export const YellowRibbonApp = ({ results }) => (
                 you get varies by school, degree type, and the program
                 you&apos;re enrolled in.
               </p>
+
               <a
                 href="/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/"
                 rel="noopener noreferrer"
               >
                 Find out if you qualify for the Yellow Ribbon Program.
               </a>
+
               <p>
                 If you don&apos;t already have Post-9/11 GI Bill benefits, you
                 can:
               </p>
+
               <ul>
                 <li>
                   <a href="/education/eligibility/" rel="noreferrer noopener">
                     Find out if you&apos;re eligible for the Post-9/11 GI Bill
                   </a>
                 </li>
+
```

Comment:
Interesting change. How come you used 
```

```

Instead of 

```

```

?

Reply:
well I thought it would be better than:
```

```

Reply:
Does that pass unit tests?

Reply:
Almost... it wanted to plan a space party for the unit test but didn't know how. Eventually, we figured out how to planet.

Reply:
Oh wow, well that is certainly one way to go about it.